### DISCUSSION TITLE
EdgePhoto | New feature introduced in Microsoft Edge canary version 116.0.1934.0

### DISCUSSION THREAD

**Discussion by HotCakeX** [SpyNetGirl/MSEdgeFeatures#4] [Announcements]
This feature allows Edge to become a Photo viewer app. This is the initial appearance of the feature so things aren't even finished yet, just gives you an idea of what is coming. 

Video demo:
https://www.youtube.com/watch?v=tkAdafp9VRM

**Comment by JenieX** [comment]
That is very cool.

**Comment by sameerasw** [comment]
Hope this and the native pdf viewer gets the ability to automatically open files as PWA windows.

**Comment by JenieX** [comment]
Seems like you can access this feature by navigating to something like:
```
edge://photo/?file:///D:/gallery\image-1.png
```

I noticed that after disabling the flag.

**Reply by HotCakeX** [reply]
Nice, i just tried it too!
